nuklear.h

```diff
@@ -19798,7 +19798,10 @@ nk_panel_end(struct nk_context *ctx)
         {
             /* horizontal scrollbar */
             nk_flags state = 0;
-            scroll.x = layout->bounds.x;
+            if(layout->flags & NK_WINDOW_SCALE_LEFT)
+                scroll.x = layout->bounds.x + scrollbar_size.x + panel_padding.x * 0.5;
```

Comment:
Please add this comment and we can merge it:

>can't scale otherwise (horizontal scrollbar has priority) => reposition horizontal scrollbar to avoid overlap with scaler

Reply:
Something like this? https://github.com/Immediate-Mode-UI/Nuklear/pull/371/files#r772458292

Reply:
Not quite there yet IMHO. The rewording of yours suggests that the horizontal scrollbar can't scale. But actually it's not about the horizontal scrollbar "bar" but about the scaling "button" ("scaler") next to the scrollbar.

Or maybe I'm too tired today...